CDTDatastoreTests/CDTReplicationTests.m

```diff
@@ -401,20 +332,17 @@ -(void)testCreatePushReplicationWithFilter
     XCTAssertNotNil(replicator, @"%@", push);
     XCTAssertNil(error, @"%@", error);
 
-    NSDictionary *pushDoc = [push dictionaryForReplicatorDocument:nil];
+    NSMutableDictionary *pushDoc = [[NSMutableDictionary alloc] init];
```

Comment:
Rewrite this test without reference to `pushDoc`. There's actually not much to do in this test any more, but you could check that the underlying `TDReplicator` has had the correct filter and params set from the `CDTReplicator` object.